students/uss_tomasz/lesson_01_basics/nowy.py

```diff
@@ -0,0 +1,114 @@
+from math import ceil
```

Comment:
lesson 1 should not be the part of this PR
you probably branched out lesson_02_jues_jedzej from lesson_01_jues_jedzej
you should branch out from master branch instead. You can fix this by rebasing your branch on top of master and pushing the branch again with force